sample-notificator-app/src/main/groovy/life/qbic/samplenotificator/components/email/html/SendmailHeader.java

```diff
@@ -0,0 +1,37 @@
+package life.qbic.samplenotificator.components.email.html;
+
+/**
+ * <p>A representation of the email header necessary for the sendmail command</p>
+ */
+public class SendmailHeader {
+  String from = "noreply@qbic.life";
+  String subject = "Sample Status Update";
+  String contentType = "text/html";
+
+  SendmailHeader withSubject(String subject) {
+    this.subject = subject;
+    return this;
+  }
+  /**
+   * Creates a multi-line String with the values of this header.
+   *
+   * <pre>
+   *   From: noreply@qbic.life
+   *   Subject: Sample Status Update
```

Comment:
This will not be correct if you change the subject (withSubject()). I am confused now if this is formatting of fixed values of you create it here.

Reply:
Yes this is only an example 😄